lib/elixir/lib/io/ansi/docs.ex

```diff
@@ -524,7 +524,7 @@ defmodule IO.ANSI.Docs do
   end
 
   defp escape_underlines_in_link(text) do
-    ~r{https?\S*}
+    ~r{https?://\S*}
```

Comment:
I am not with my computer so I cannot fully read the code, but links should not be limitted to the http protocol

Reply:
This PR is not changing this though, so that's another discussion.

Reply:
I can have a look at this later.